CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 18.2.0 - 2021-10-29
+This release includes several bug fixes, introduces Klarna as a payment method binding, and renables wechat pay within the SDK
```

Comment:
could you add a link to `wechat pay` with a link?
`[WeChat Pay](https://github.com/stripe/stripe-android/tree/master/wechatpay)`